src/sphinx/code/SqlToSlick.scala

```diff
@@ -43,7 +43,7 @@ object SqlToSlick extends App {
   val db = Database.forConfig("h2mem1")
   try {
 
-    Await.result(db.run(Action.seq(
+    Await.result(db.run(DBIO.seq(
```

Comment:
`fold` and `sequence` are the standard monadic combinators, the same as on `Future`. `seq` is the simplest and most commonly used (in my experience of rewriting the tests) combinator, therefore the short name and the varargs. It is functionally equivalent to `sequence` except that it discards the return value.